mesmerize_napari/main_offline_gui.py

```diff
@@ -367,17 +369,24 @@ def view_projections(self):
         projection = s.caiman.get_projection(proj_type=proj_type)
         self.viewer.add_image(projection, name=f'{proj_type}: projection {s["name"]}', colormap='gnuplot2')
 
-    def downsample_mcorr(self):
-        # s = self.selected_series()
-        # algo = s['algo']
-        # if algo == 'mcorr':
-        #     output_path = s.mcorr.get_output_path()
-        #     self._open_movie(output_path, name=f'mcorr: {s["name"]}')
-        #     Yr, dims, T = cm.load_memmap(self.input_movie_path)
-        #     images = np.reshape(Yr.T, [T] + list(dims), order='F')
-        #     self.viewer.add_image(images, name=name, colormap='gnuplot2')
-        pass
+    def view_downsample_mcorr(self):
+        s = self.selected_series()
+        downsample_ratio = self.ui.spinBoxDownsampleRatio.value()
+        images = s.mcorr.get_output()[::downsample_ratio,:,:]
```

Comment:
I should see how fast this is for a big movie

Reply:
OK it's fast

Reply:
The minimum value for the spinBox widget for this should be 1, zero makes no sense. Set the default value as 10?

Reply:
Default I'll set as 4, that's more reasonable generally

Reply:
change this to something averaging frames within a window